backend/src/lib/api-docs/constants.ts

```diff
@@ -674,7 +674,9 @@ export const INTEGRATION = {
       secretGCPLabel: "The label for GCP secrets.",
       secretAWSTag: "The tags for AWS secrets.",
       kmsKeyId: "The ID of the encryption key from AWS KMS.",
-      shouldDisableDelete: "The flag to disable deletion of secrets in AWS Parameter Store."
+      shouldDisableDelete: "The flag to disable deletion of secrets in AWS Parameter Store.",
```

Comment:
This isn't being used anywhere in this PR but it has been added?

Reply:
this is just a formatting update, a comma was added